tests/unittests/OCLTest.cpp

```diff
@@ -34,8 +34,8 @@ TEST(OpenCLCorrectnessTest, convOps) {
   Tensor out1;
   Tensor out2;
 
-  inferBasicConvNet(&inputs, &out1, BackendKind::OpenCL, 4);
-  inferBasicConvNet(&inputs, &out2, BackendKind::Interpreter, 4);
+  inferBasicConvNet(&inputs, &out1, BackendKind::OpenCL, 64);
```

Comment:
we have all these tests in backend correctness tests. Maybe it's time to remove this OCLTest